.evergreen/run-mongodb.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -o xtrace
+
+# User configurable-options
+export MONGODB_VERSION="4.2"
+export TOPOLOGY="replica_set"
```

Comment:
why run against a replicaset? I don't think we're testing anything complicated, I would just start up a single server

Reply:
Changed to standalone.